modules/ROOT/pages/access-client-app-id-task.adoc

```diff
@@ -14,7 +16,8 @@ To obtain the credentials of a client application:
 . From the details pane, click the application for which you want to view the client credentials.
 +
 The client ID and client secret credentials for the selected application appear on the right of the window. 
-. Alternatively, if you need to obtain only the client ID of the registered client application, select the correct version of the API and click *Contracts* from the left navigation. You can then click the specific API contract to view the details. 
+. Alternatively, if you need to obtain only the client ID of the registered client application, select the correct version of the API and click *Contracts* from the left navigation. 
```

Comment:
It may be useful to add as related this Doc
https://docs.mulesoft.com/exchange/about-my-applications

If the user does not have access to API Manager, but he is the Application Owner, he can see the Application Credentials in Exchange.

Reply:
Done! Thanks!